### ISSUE TITLE
Create a Provider Class example raises an exception

### ISSUE THREAD

**Issue by radekholy24** [jazzband/django-payments#405]
The example in the [Create a Provider Class](https://django-payments.readthedocs.io/en/latest/backends.html#create-a-provider-class) section of the documentation does not work.

Reproducer:

1. Copy the example from the [Create a Provider Class](https://django-payments.readthedocs.io/en/latest/backends.html#create-a-provider-class) section of the documentation
2. Implement the `process_data` and `get_form` methods
3. Run an app that uses the provider

Actual behavior:

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'payments.providers'
```

Expected behavior:

The app is integrated with my payment provider.

**Comment by Abhishek-jha-96** [comment]
Though you are correct there is no 'payments.providers' module. one thing you can try is use:
from payments.core import BasicProvider

i think this might will serve the purpose.

**Comment by radekholy24** [comment]
Yes, `from payments.core import BasicProvider` works. Could you fix the example, please?

**Comment by Abhishek-jha-96** [comment]
Sure
